modules/creating-a-runtime-configuration.adoc

```diff
@@ -51,7 +51,7 @@ You can obtain the data science pipelines API endpoint from the *Data science pi
 +
 [IMPORTANT]
 ====
-If you created a notebook directly from the *Start basic workbench* tile on the dashboard, select `EXISTING_BEARER_TOKEN` from the *Authentication Type* list.
```

Comment:
8. Mixed Authentication Context
File: modules/creating-a-runtime-configuration.adoc
Issue: The EXISTING_BEARER_TOKEN instruction is specifically for basic workbenches but could be clearer about when this applies vs. other authentication types.

@coderabbitai auth stuff goes over my head, especially without reading more of the context; you may be right, or you may be not; i am not going to investigate which